t/unit/utils/test_time.py

```diff
@@ -131,6 +132,8 @@ def test_remaining():
     """
     eastern_tz = ZoneInfo("US/Eastern")
     tokyo_tz = ZoneInfo("Asia/Tokyo")
+    eastern_tz_pytz = pytz.timezone("US/Eastern")
```

Comment:
instead of pytz, shouldn't we use ZoneInfo?

Reply:
https://stackoverflow.com/questions/70249473/convert-an-aware-datetimes-timezone-to-utc-with-zoneinfo-migration-away-from-p